packages/service-clients/end-to-end-tests/odsp-client/src/test/audience.spec.ts

```diff
@@ -24,14 +24,27 @@ describe("Fluid audience", () => {
 	const connectTimeoutMs = 10_000;
 	let client: OdspClient;
 	let schema: ContainerSchema;
+
+	const credentials = getCredentials();
+
+	if (credentials.length < 2) {
+		throw new Error("Insufficient login credentials provided");
+	}
+
+	const [cred1, cred2] = credentials.slice(0, 2);
```

Comment:
Don't need to slice here, can just destructure the original array.
```suggestion
	const [cred1, cred2] = credentials;
```